troubleshooting/tools/log-loss-test-framework/awslogs-benchmarking-scripts/fargate_launcher.sh

```diff
@@ -0,0 +1,57 @@
+export TASK_DEF_FILE=fargate-task.json
+export RUN_TASK_FILE=fargate-run-task.json
+export TOTAL_SIZE_IN_MB="1000"
+export REGION='us-west-2'
+export THROUGHPUTS="
+1000
+2000
+3000
+4000
+4500
+5000
+6000
+7000
+8000
+9000
+10000
+12000
+"
+export BUFFER_SIZES="
+1m
+2m
+4m
+6m
+8m
+12m
+"
```

Comment:
Can't we move those into util class, why it needs to be redefined here ??  Since `ecs_launcher.sh` has same variables

Reply:
this is a simple quick script to make launching easy for the AWSLogs project... it may never be used again... so I think its not worth it